c8/tenant.py

```diff
@@ -606,7 +607,7 @@ def get_database_access_level_user(self, username, databasename=""):
         def response_handler(resp):
             if resp.is_success:
                return resp.body['result']
-            raise DataBaseError(resp, request)
+            raise GetDataBaseAccessLevel(resp, request)
```

Comment:
I would add an else clause here. What do you think? We are raising an error if the response is NOK.

Reply:
Yes, we are raising an exception if the response is not a success (i.e. it contains a field called error: True). So I don't think an else is needed here

Reply:
Great! Thanks